src/core/file_sys/directories/normal_directory.cpp

```diff
@@ -0,0 +1,139 @@
+// SPDX-FileCopyrightText: Copyright 2025 shadPS4 Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "common/alignment.h"
+#include "common/assert.h"
+#include "common/logging/log.h"
+#include "common/singleton.h"
+#include "core/file_sys/fs.h"
+#include "normal_directory.h"
```

Comment:
```suggestion
#include "core/file_sys/directories/normal_directory.h"
```
Adjust in other files as well to have full path for consistency